README.md

```diff
@@ -29,8 +43,16 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+function Instructor(name) {
+  var self = this;
+  this.name = name;
+  this.givesHomework = function(assignment) {
+    console.log(self.name + " gives the students " + assignment + " for Friday's homework.")
```

Comment:
Great! In this case you don't have to use the `var self = this` since this would still be referring to the instance created using the constructor function. This would work the same: `console.log(this.name + " gives the students " + assignment + " for Friday's homework.");`